pairing/bn256/suite.go

```diff
@@ -98,6 +98,10 @@ func (s *Suite) Pair(p1 kyber.Point, p2 kyber.Point) kyber.Point {
 	return s.GT().Point().(*pointGT).Pair(p1, p2)
 }
 
+func (s *Suite) ValidatePairing(p1, p2, inv1, inv2 kyber.Point) bool {
```

Comment:
is there a naive slower way to do this generally you could put here instead of the panic?

Reply:
Added comment in the interface definition